Gemfile

```diff
@@ -91,14 +91,13 @@ gem "rack-throttle"
 gem "secure_headers", "~> 2.5.0"
 
 # Frontend
-gem "backbone-on-rails", "~>0.9.10.0" # Legacy js
```

Comment:
This blocks upgrade - but will be removed as soon as we finish the admin update. So it's coming soon.

Reply:
Resolved by #1146